nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-                      "value": 8.0
-                    },
-                    {
-                      "year": 2001,
-                      "value": 8.1
-                    },
-                    {
-                      "year": 2002,
-                      "value": 8.2
-                    },
-                    {
-                      "year": 2003,
-                      "value": 8.3
-                    },
-                    {
-                      "year": 2004,
-                      "value": 9.0
-                    },
-                    {
-                      "year": 2005,
-                      "value": 10.5
-                    }
-                  ]
-                },
-                {
-                  "label": "Haziest Days",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 8.0
-                    },
-                    {
-                      "year": 2001,
-                      "value": 8.1
-                    },
-                    {
-                      "year": 2002,
-                      "value": 8.2
-                    },
-                    {
-                      "year": 2003,
-                      "value": 8.3
-                    },
-                    {
-                      "year": 2004,
-                      "value": 9.0
-                    },
-                    {
-                      "year": 2005,
-                      "value": 10.5
-                    }
-                  ]
-                }
-              ],
-              "trendLines": [
-                {
-                  "year": 2000,
-                  "lines": [
-                    {
-                      "label": "Theil trend line slope = -0.18 dv/yr significant trend",
-                      "slope": -0.18,
-                      "pvalue": 0.05
-                    },
-                    {
-                      "label": "Some other trend line",
-                      "slope": 2,
-                      "pvalue": 1
-                    }
-                  ]
-                },
-                {
-                  "year": 2001,
-                  "lines": [
-                    {
-                      "label": "Theil trend line slope = -0.5 dv/yr significant trend",
-                      "slope": -0.5,
-                      "pvalue": 0.2
-                    },
-                    {
-                      "label": "Some other trend line",
-                      "slope": 2,
-                      "pvalue": 1
-                    }
-                  ]
-                }
-              ],
-              "constantLines": [
-                {
-                  "label": "Natural Condition, Haziest Days",
-                  "value": 12
-                },
-                {
-                  "label": "Natural Condition, Clearest Days",
-                  "value": 5
-                }
-              ]
-            },
-            {
```

Comment:
Each member of this array represents another one of these charts:
![screen shot 2018-11-09 at 1 04 09 pm](https://user-images.githubusercontent.com/3201135/48279901-fcba7f80-e41f-11e8-8eac-3ddfcf59b273.png)
With all the self-contained data needed to draw it for any given end-year and time-period chosen.

Reply:
Okay.